packages/ChangesReloaded-Core.package/CRBrowser.class/instance/createWindowSpecWith..st

```diff
@@ -0,0 +1,11 @@
+building
+createWindowSpecWith: aBuilder
+
+	| windowSpec |
+	windowSpec := aBuilder pluggableWindowSpec new
+		model: self;
+		label: 'Changes Browser';
+		extent: self extent;
+		"closeAction: self delete;"
```

Comment:
well  that comment is there because we feel the same as you about the actual delete method

Reply:
Okay. So if I remember correctly, `closeAction` will be executed when the PluggableSystemWindow is about to be closed and deleted. It's a clean up method for everything inside the window. Since we want the CHBrowser to be deleted when the window closes, I think this code shouldn't be a comment

Reply:
Ok I misphrased or made that not clear enough - yes you are right when it comes to the closeAction but this is not what we didn't know. The point is that self delete is actually a method that we/legacy people implemented that we (and you, see discussion below) don't understand. The CHBrowser will be deleted either way, a more correct way to call the called method would be onDelete

Reply:
happy to hear more about it in the in-person review ^^